package.json

```diff
@@ -33,6 +33,7 @@
     ]
   },
   "dependencies": {
+    "babel-preset-env": "^1.6.0",
```

Comment:
How is this module useful?

Reply:
it was used before with "await" promise syntax, since that code was changed by a more traditional style code, that dependency is no longer needed. removed